dev-only.js

```diff
@@ -1,13 +1,12 @@
 'use strict';
-const isNode = require('is-node');
 
-if (process.env.NODE_ENV === 'production' && !isNode) {
```

Comment:
Note: The reason I did this is that the `isNode` check caused various problems for me when using Parcel, as well as when compiling `ow` using `npm run build`. It also seems more intuitive to me that if I require `ow/dev-only`, it won't be bundled when running in production, even if I'm running on a Node-like environment.